docs/_documentations/mdt-vsc-troubleshooting.md

```diff
@@ -18,6 +18,18 @@ You can also post questions on our [Mattermost channel](https://mattermost.eclip
 
 ***
 
+### **Codewind Output stream**
+
+The Codewind Output stream is a tab within Codewind that is visible in the VS Code editor. It logs `cwctl` commands together with their ouput, for example:
+
+![Image of VS Code output](dist/images/cdt-vsc/output.png)<br>
```

Comment:
I don't think we need both images, we can just show the second one.

Reply:
Removed the first, and tweaked the words accordingly.